### PR TITLE
Feautr/#10

### PR DESCRIPTION
**Pull request by FirdavsAsadov**
ish tugadi

### CODE REVIEW

**Review by Jasurbek1223 — DISMISSED** [review]

**Review by Jasurbek1223 — DISMISSED** [review]

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Enums/LanguageEnum.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Enums/LanguageEnum.cs]: Clean all useless namespaces

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Enums/LanguageEnum.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+namespace TestGorilla.Domain.Enums
+{
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Enums/LanguageEnum.cs:3]: Make a clean code

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/CheckboxQuestion.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestGorilla.Domain.Models.Questions.InterfeysQuestion;
+
+namespace TestGorilla.Domain.Models.Questions
+{
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/CheckboxQuestion.cs]: Clean your code here too

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/Interfeys/IQuestion.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestGorilla.Domain.Models.Questions.InterfeysQuestion
+{
+    public interface IQuestion
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/Interfeys/IQuestion.cs]: Clean your code

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Test.cs [hunk 5]
```diff
@@ -3,10 +3,16 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using TestGorilla.Domain.Enums;
 
 namespace TestGorilla.Domain.Models
 {
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Test.cs:5]: Clean your code

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/ShortQuestion.cs [hunk 6]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestGorilla.Domain.Models.Questions.InterfeysQuestion;
+
+namespace TestGorilla.Domain.Models.Questions
+{
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Questions/ShortQuestion.cs:4]: Clean your code

**Review by azam-amonov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**